protocol/messenger_messages_tracking_test.go

```diff
@@ -172,7 +172,7 @@ func (s *MessengerMessagesTrackingSuite) testMessageMarkedAsSent(textSize int) {
 	// Message should be marked as sent eventually
 	err = tt.RetryWithBackOff(func() error {
 		rawMessage, err = s.bob.persistence.RawMessageByID(inputMessage.ID)
-		if err != nil || !rawMessage.Sent {
+		if err != nil || rawMessage.SendCount < 1 {
```

Comment:
Is `Sent == SendCount > 0`? :thinking:

Reply:
It's a workaround that it's not able to query the store node in tests, only the successful query to store node will mark the message as sent. @osmaczko

Reply:
Thanks, I got the reasoning. I am wondering, would it require significant effort to either make the requirement for messages being received by store nodes configurable or to mock the store node response, allowing this test to remain valid?

Reply:
Thanks for details. I don't evaluate it to be high priority, let's wait for nwaku and see what we can do.